### PR TITLE
Update readme, automatically open the browser, made it work in maven …

### PR DESCRIPTION
**Pull request by marcantoineveilleux**
Update readme, automatically open the browser in dev mode, made it work in mavenless environments.

Aka, trying to make it work for HG + NPM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,25 +5,83 @@
...
 
+Environment variables:
 - `IGNORE_DECLINED_PR=1` Will ignore declined PR when looking if the PR exists
 - `BITBUCKET_CLIENT_ID` the bitbucket OAuth **Key** previously created
 - `BITBUCKET_CLIENT_SECRET` the bitbucket OAuth **Secret** previously created
-- `BITBUCKET_REPO_NAME` https ://bitbucket.org/owner/**name**
-- `BITBUCKET_REPO_OWNER` https ://bitbucket.org/**owner**/name
 - `DRY_RUN` won't create a PR
 
+(Coveo employees take a look to https://jenkins.cloud.coveo.com/job/Reveal_Lure )
```
Comment (AndyCoveo) [inline README.md]: I don't think this should be included here.

File: .vscode/launch.json.example [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+                "DRY_RUN": 1,
+                "BITBUCKET_CLIENT_ID": "yourid",
+                "BITBUCKET_CLIENT_SECRET": "yoursecret",
+                "LURE_AUTO_OPEN_AUTH_PAGE": "1"
+            },
+            "args": [
+                "-auth",
+                "oauth",
+                "-config",
+                "/Users/maveilleux/Repositories/SalesforceIntegrationV2/1/lure.config"
```
Comment (AndyCoveo) [inline .vscode/launch.json.example]: maveilleux :)

Comment (dreisch-coveo) [inline .vscode/launch.json.example]: 👀

File: README.md [hunk 3]
```diff
@@ -5,25 +5,83 @@
...
+First create a `lure.config` in the repository you want to keep your dependencies up-to-date.
+
+The file should look like:
+
+```{
+    "projects": [
+        {
+            "vcs": "hg",
+            "owner": "dreisch",
+            "name": "catfeederhg",
```
Comment (dreisch-coveo) [inline README.md:19]: 👓

File: README.md (hunk 1 above)
Comment (dreisch-coveo) [inline README.md]: 👀 
well maybe who cares about the url but "reveal" is pretty forbidden for sure

Comment (marcantoineveilleux) [reply README.md]: What do you mean... the jenkin's job purpose is to _reveal_ how to use lure. :trollface:

**Comment by fredboutin** [comment]
Nice stuff, thanks for the readme update 🙏. Fix the comments others wrote and I'm good with that.

**Review by fredboutin — APPROVED** [review]

**Review by dreisch-coveo — APPROVED** [review]
Don't forget to either rebase or somehow squash the commits so we don't have "Code review." in the commit history.

**Pull request merged**